tools/migrations/043_modify_choco_colour_column.py

```diff
@@ -0,0 +1,24 @@
+import mariadb
+
+
+def migration_name():
+    return "Rename colour column to color in char_chocobos"
+
+
+def check_preconditions(cur):
+    return
+
+
+def needs_to_run(cur):
+    cur.execute("SHOW COLUMNS FROM char_chocobos ;")
```

Comment:
This looks incomplete, should check if a colour column exists.

Reply:
Colour or color? 

`def needs_to_run(cur):
    # Ensure column doesn't already exist.
    cur.execute("SHOW COLUMNS FROM char_chocobos LIKE 'color';")

    row = cur.fetchone()
    if row:
        return False

    return True`

Reply:
Don't think it matters, either way should be fine.